src/main/java/thePackmaster/cards/pinnaclepack/TopOfTheSkyUncommonAttack.java

```diff
@@ -0,0 +1,47 @@
+package thePackmaster.cards.pinnaclepack;
+
+import com.evacipated.cardcrawl.mod.stslib.variables.ExhaustiveVariable;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.vfx.combat.IronWaveEffect;
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class TopOfTheSkyUncommonAttack extends AbstractPinnacleCard {
```

Comment:
*Top of the Sky*
The overall design of this card is okay (though it's another card that runs right up to the text length limits), but the upgrade is out of line with how good it is -- considering what you get if you play the card until it exhausts, it goes from 4 damage/2 block to 9 damage/6 block. This is due to the design of returning to your hand when you play it, which chanages how we have to evaluate the card.

The cleanest ways to simplify this are to either (a) remove the increase to exhaustive or (b) remove the upgrade to damage/block.